### PR TITLE
:book: Update troubleshooting.md

### PR DESCRIPTION
**Pull request by christianh814**
Added an example of a "one liner" to label workers

**What this PR does / why we need it**:

Sample "one liner" to label nodes to get people "up and running" faster

### CODE REVIEW

File: docs/book/src/user/troubleshooting.md [hunk 1]
```diff
@@ -13,3 +13,9 @@ Assigning such labels to Nodes must be done after the bootstrap process has comp
 ```
 kubectl label nodes <name> node-role.kubernetes.io/worker=""
 ```
+
+For convenience, here is an example of a "one linner" to do this post installation
```
Comment (vincepri) [inline docs/book/src/user/troubleshooting.md]:
```suggestion
For convenience, here is an example one-liner to do this post installation 
```

Comment (christianh814) [reply docs/book/src/user/troubleshooting.md]: I must have been hungry! :laughing:

**Comment by christianh814** [comment]
I signed it

**Comment by chuckha** [comment]
/approve
/lgtm

```
chuckh-a02:kubeadm cha$ k get nodes
NAME                 STATUS   ROLES    AGE   VERSION
kind-control-plane   Ready    master   76s   v1.17.0
kind-worker          Ready    <none>   35s   v1.17.0
kind-worker2         Ready    <none>   43s   v1.17.0
kind-worker3         Ready    <none>   45s   v1.17.0
chuckh-a02:kubeadm cha$ kubectl get nodes --no-headers -l '!node-role.kubernetes.io/master' -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}' | xargs -I{} kubectl label node {} node-role.kubernetes.io/worker=''
node/kind-worker labeled
node/kind-worker2 labeled
node/kind-worker3 labeled
chuckh-a02:kubeadm cha$ k get nodes
NAME                 STATUS   ROLES    AGE   VERSION
kind-control-plane   Ready    master   80s   v1.17.0
kind-worker          Ready    worker   39s   v1.17.0
kind-worker2         Ready    worker   47s   v1.17.0
kind-worker3         Ready    worker   49s   v1.17.0
chuckh-a02:kubeadm cha$ 
```

works well!

**Pull request merged**